index.html

```diff
@@ -60,19 +60,12 @@
       >
         <div class="contact-info d-flex align-items-center">
           <i class="bi bi-phone d-flex align-items-center"
-            ><span>+91 9876543210</span></i
+            ><span>+91 9876782011</span></i
```

Comment:
We should probably also add hyphens (-) somewhere in the number. Take reference from other websites on how to print an Indian phone number.

Reply:
hanji i added hyphen instead of space